config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits-main.yaml

```diff
@@ -268,32 +244,31 @@ presubmits:
         - ./hack/e2e.sh
         env:
         - name: GINKGO_SKIP
-          value: "\\[clusterctl-Upgrade\\] \\[specialized-infra\\] \\[Conformance\\]"
+          value: "\\[Conformance\\] \\[specialized-infra\\]"
         # we need privileged mode in order to do docker in docker
         securityContext:
           privileged: true
           capabilities:
-            add: ["NET_ADMIN"]
+            add: [ "NET_ADMIN" ]
         resources:
           requests:
             cpu: "4000m"
             memory: "6Gi"
     annotations:
       testgrid-dashboards: vmware-cluster-api-provider-vsphere, sig-cluster-lifecycle-cluster-api-provider-vsphere
-      testgrid-tab-name: pr-full-e2e
+      testgrid-tab-name: pr-e2e-full-main
       testgrid-alert-email: sig-cluster-lifecycle-cluster-api-vsphere-alerts@kubernetes.io
       description: Runs all e2e tests
 
-  - name: pull-cluster-api-provider-vsphere-conformance
+  - name: pull-cluster-api-provider-vsphere-conformance-main
     branches:
-      - ^main$
+    - ^main$
     labels:
       preset-dind-enabled: "true"
       preset-cluster-api-provider-vsphere-e2e-config: "true"
       preset-cluster-api-provider-vsphere-gcs-creds: "true"
       preset-kind-volume-mounts: "true"
     always_run: false
```

Comment:
Dropped the optional to end up with the same behavior as for e2e-full. Basically it should be mandatory for merge as soon as someone manually triggered the job